modules/common/src/main/java/org/opencastproject/security/api/JaxbRole.java

```diff
@@ -45,10 +47,12 @@
   @XmlElement(name = "description")
   protected String description;
 
-  /** The description */
   @XmlElement(name = "organization")
   protected JaxbOrganization organization;
 
+  @XmlElement(name = "organizationid")
```

Comment:
Maybe organization_id or organizationId for sake of readability?

Reply:
We have several precedents to choose from :-)

```
@XmlElement(name = "warning_state_trigger")
@XmlElement(name = "processingHost")
@XmlElement(name = "samplingrate")
```

but of these, camelCase seems to be the more common style used in our code base so far.

Reply:
In #763, I've switched to `organization-id` because of `execution-history` and `time-in-queue`, `fail-on-error`, `retry-strategy`, … it seemed to me that `-` was commonly used ;-D

I tend to use it here as well unless someone wants something else in particular.

Reply:
For me, organization-id is fine.

Reply:
Sounds good.

Reply:
@lkiesow #763 is merged, so let's go for organization-id ...